charts/chronicle/Chart.yaml

```diff
@@ -17,16 +17,16 @@ keywords:
 # This is the chart version. This version number should be incremented each
 # time you make changes to the chart and its templates, including the app
 # version.
-version: 0.1.7
+version: 0.1.13
 
 # This is the version number of Chronicle being deployed. This version
 # number should be incremented each time you make changes to Chronicle.
-appVersion: 0.2.1
+appVersion: 0.6.2
 
 dependencies:
   - name: standard-defs
     version: ~0.1.0
     repository: https://btp-charts-stable.s3.amazonaws.com/charts/
   - name: sawtooth
     version: ~0.2.0
-    repository: https://btp-charts-stable.s3.amazonaws.com/charts/
+    repository: https://btp-charts-unstable.s3.amazonaws.com/charts/
```

Comment:
This should stay pointing to btp-stable. In fact we should probably update the chronicle-on-sawtooth chart to point to stable as well